astropy/units/function/logarithmic.py

```diff
@@ -150,6 +150,21 @@ def _default_function_unit(self):
     def _quantity_class(self):
         return Dex
 
+    def to_string(self, format = 'generic'):
+        if format == 'cds':
+            return "[{}]".format(self.physical_unit.to_string(format=format))
+        else:
+            self_str = self.function_unit.to_string(format)
```

Comment:
No need to repeat code from above; as I noted in the original comment, you can just do
```
return super().to_string(format=format)
```